src/arfsdao.ts

```diff
@@ -276,9 +276,6 @@ export class ArFSDAOAnonymous extends ArFSDAOType {
 		maxDepth: number,
 		includeRoot: boolean
 	): Promise<ArFSPublicFileOrFolderWithPaths[]> {
-		if (maxDepth < 0 || !Number.isInteger(maxDepth)) {
```

Comment:
Why remove these guards? If you had left them in, you'd have seen that using POSITIVE_INFINITY doesn't actually work 🤔

Reply:
While that benefits the CLI layer, it removes the safety checks here for ArDrive Core itself (once ArFSDao ultimately lives there).